test/Interop/Cxx/operators/non-member-out-of-line-irgen.swift

```diff
@@ -4,5 +4,5 @@ import NonMemberOutOfLine
 
 public func add(_ lhs: IntBox, _ rhs: IntBox) -> IntBox { lhs + rhs }
 
-// CHECK: call i32 @_Zpl6IntBoxS_(i32 %{{[0-9]+}}, i32 %{{[0-9]+}})
-// CHECK: declare i32 @_Zpl6IntBoxS_(i32, i32)
+// CHECK: call i32 [[NAME:@.*]](i32 %{{[0-9]+}}, i32 %{{[0-9]+}})
+// CHECK: declare i32 [[NAME]](i32, i32)
```

Comment:
This makes it a rather generic test -- I'd rather prefer that it looked more like `test/Interop/Cxx/extern-var/extern-var-irgen.swift`.

Reply:
How about now?

Reply:
👍